e2e/cypress/integration/messaging/system_message_limited_options_spec.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('sysadmin');
+        cy.visit('/');
+    });
+
+    it('M17458 - System message limited options', () => {
+        // # Update channel header to create a new system message
+        cy.updateChannelHeader(Date.now());
+
+        // # Get system message Id
+        cy.getLastPostId().then((lastPostId) => {
+            // # Mouse over the post to show the options
+            cy.get(`#post_${lastPostId}`).trigger('mouseover', {force: true});
+            cy.wait(TIMEOUTS.TINY);
+
+            // * No option to reply this post
+            cy.get(`#CENTER_commentIcon_${lastPostId}`).should('not.exist');
+
+            // * No option to react to this post
+            cy.get(`#CENTER_reaction_${lastPostId}`).should('not.exist');
+
+            // # Click in the '...' button
+            cy.get(`#CENTER_button_${lastPostId}`).click({force: true});
+            cy.wait(TIMEOUTS.TINY);
+
+            // # Get all list elements in the dropdown
+            cy.get(`#CENTER_dropdown_${lastPostId}`).find('li').then((items) => {
+                // * Must be only 1 element
+                expect(items.length).to.equal(1);
+
+                // * The element must be delete
+                expect(items[0].id).to.equal(`delete_post_${lastPostId}`);
+            });
+
+            // # Log-in as a different user
+            cy.apiLogin('user-1');
+            cy.visit('/');
+
+            // # Mouse over the post to show the options
+            cy.get(`#post_${lastPostId}`).trigger('mouseover', {force: true});
```

Comment:
The testcase expects the user-1 to be on the same team and same channel as sysadmin, which could not be the case. The test fails because of it as shown in the screenshot attached. Can you please update the steps so that both the users access the same team and channel to avoid this issue. 
![Screenshot 2019-12-11 at 10 03 57 PM](https://user-images.githubusercontent.com/1429138/70640654-48161180-1c62-11ea-8703-2b96cc52c57e.png)

Reply:
Thanks for the review @srkgupta . I am not sure if the screenshot is to clarify the possible problem, or it is just mistaken from a different review. Either way, I made a small change to make sure both users are on the same channel (based on base information provided on make test-data).